tools/find-inactive-collaborators.mjs

```diff
@@ -69,14 +75,22 @@ async function retrieveCollaboratorsFromReadme() {
       break;
     }
     if (line.startsWith('**') && isCollaborator) {
-      returnedArray.push(line.split('**', 2)[1].trim());
+      const [_, name, email] = /^\*\*([^*]+)\*\* &lt;(.+)&gt;/.exec(line);
```

Comment:
To fix the linter error
```suggestion
      const [, name, email] = /^\*\*([^*]+)\*\* &lt;(.+)&gt;/.exec(line);
```

Reply:
I thought I ran `make lint-js` before pushing. Guess not. 🙃